src/viam/sdk/components/private/board_server.cpp

```diff
@@ -174,6 +177,51 @@ ::grpc::Status BoardServer::GetDigitalInterruptValue(
     return ::grpc::Status();
 }
 
+::grpc::Status BoardServer::StreamTicks(
```

Comment:
Is there something that prevents use of `make_service_helper` here? Thoughts much like my request about the client helper.

Reply:
changed to use it here.

Reply:
I think there is a bunch of code that can now be removed here, then.